oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/server/WebSessionServerOAuth2AuthorizedClientRepository.java

```diff
@@ -94,14 +94,10 @@ public Mono<Void> removeAuthorizedClient(String clientRegistrationId, Authentica
 		// @formatter:on
 	}
 
-	@SuppressWarnings("unchecked")
```

Comment:
I don't think this should be removed. Can you restore this line?

Reply:
Apologies as you were correct to remove this. I'll re-remove it in a polish commit.